upload_logs.py

```diff
@@ -29,15 +31,24 @@
     pipeline_configuration_file_path = args.pipeline_configuration_file_path
     run_id = args.run_id
     memory_profile_file_path = args.memory_profile_file_path
+    data_archive_file_path = args.data_archive_file_path
 
     log.info("Loading Pipeline Configuration File...")
     with open(pipeline_configuration_file_path) as f:
         pipeline_configuration = PipelineConfiguration.from_configuration_file(f)
-        
+    '''        
```

Comment:
Why is this commented out?

Reply:
Forgot this here. Thanks for catching it.